deployment/hasura/migrations/Aerie/4_mission_model_default_view/up.sql

```diff
@@ -0,0 +1,41 @@
+create table ui.view_to_mission_model (
+  created_at timestamptz not null default now(),
+  id integer generated always as identity,
+  mission_model_id integer not null,
+  updated_at timestamptz not null default now(),
+  view_id integer not null,
+
+  constraint view_to_mission_model_primary_key
+    primary key (id),
+
+  constraint one_view_per_mission_model
+    unique (mission_model_id, view_id),
+
+  foreign key (mission_model_id)
+    references merlin.mission_model (id)
+    on delete cascade,
+
+  foreign key (view_id)
+    references ui.view (id)
+    on delete cascade
+);
```

Comment:
Since this is an association table, a better design would be to scrap the auto-generated `id` and upgrade what's currently listed as the `unique` constraint as the pkey, like what we do for tags. Additionally, association tables can't be updated, so I recommend scrapping the timestamp-related metadata:

```suggestion
create table ui.view_to_mission_model (
  mission_model_id integer not null,
  view_id integer not null,

  primary key (mission_model_id, view_id),

  foreign key (mission_model_id)
    references merlin.mission_model (id)
    on delete cascade,

  foreign key (view_id)
    references ui.view (id)
    on delete cascade
);
```